astropy/utils/misc.py

```diff
@@ -395,9 +405,7 @@ class JsonCustomEncoder(json.JSONEncoder):
 
     """
 
-    def default(self, obj):
-        import numpy as np
-
+    def default(self, obj: object) -> object:
```

Comment:
[The base class method is annotated as `def default(self, o: Any) -> Any`](https://github.com/python/typeshed/blob/a46eea77e3c5afdaa9d02a2caf17a7375547164c/stdlib/json/encoder.pyi#L38), so this can't really be annotated any better. I used `object` instead of `Any` so that if the upstream annotations are updated then we would have a chance to learn about it.